zipline/assets/_assets.pyx

```diff
@@ -251,6 +251,21 @@ cdef class Future(Asset):
         if self.end_date is None:
             self.end_date = expiration_date
 
+    property auto_close_date:
```

Comment:
@jfkirk is there a reason users can't specify this date explicitly?  I wonder if that would be a cleaner solution to @warren-oneill's need to toggle this behavior (https://github.com/quantopian/zipline/pull/672).  Then this just becomes a line in the constructor that looks like:

```
def __cinit__(self, expiration_date, notice_date, auto_close_date, ...):
    if auto_close_date is None:
        auto_close_date = min(self.notice_date, self.expiration_date)
    self.auto_close_date = auto_close_date
```
You could then effectively disable auto_closing by passing some far-future date as your "auto-close-date".  (or we could have a sentinel value that expliciltly means "never auto close".